common/serial/arch/arch_linux.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ARCH_LINUX__H
+#define ARCH_LINUX__H
+
+#include <arch.h>
+#include <stdio.h>
+
+#define TRACE() printf("%s:%d\n", __PRETTY_FUNCTION__, __LINE__)
```

Comment:
Putting it there allowed for use in any consumer of arch_linux.h. Ultimately this macro goes away, as we'll have some proper debugging class.